File: services/oauth-java/src/main/java/com/codemash/service/data/Person.java
package com.codemash.service.data;

/**
 * Created by david on 12/8/14.
 */
public class Person {
    private String FirstName;
    private String LastName;
    private String Gender;

    public String getFirstName() {
        return FirstName;
    }

    public void setFirstName(String _FirstName) {
        this.FirstName = _FirstName;
    }

    public String getLastName() {
        return LastName;
    }

    public void setLastName(String lastName) {
        LastName = lastName;
    }

    public String getGender() {
        return Gender;
    }

    public void setGender(String gender) {
        Gender = gender;
    }
}

File: ios/LoginDemo/LoginDemo/Person.swift
//
//  Person.swift
//  LoginDemo
//
//  Created by David Truxall on 12/15/14.
//  Copyright (c) 2014 Hewlett-Packard Company. All rights reserved.
//

import Foundation

class Person {

    var firstName:String
    var lastName:String
    var gender:String
    
    init(first:String,last:String,gender:String) {

        self.firstName = first
        self.lastName = last
        self.gender = gender
    }
}